playbooks/portal-report.yml

```diff
@@ -0,0 +1,400 @@
+        skynetfree_medianregistrywrite15mp99ms: "{{ skynetfree_registrywrite15mp99mslist[(skynetfree_count | int / 2) | int] }}"
+
+        # Medians skynetpro
+        skynetpro_medianbasesectorupload15mp99ms: "{{ skynetpro_basesectorupload15mp99mslist[(skynetpro_count | int / 2) | int] }}"
+        skynetpro_medianstreambufferread15mp99ms: "{{ skynetpro_streambufferread15mp99mslist[(skynetpro_count | int / 2) | int] }}"
+        skynetpro_medianregistryread15mp99ms: "{{ skynetpro_registryread15mp99mslist[(skynetpro_count | int / 2) | int] }}"
+        skynetpro_medianregistrywrite15mp99ms: "{{ skynetpro_registrywrite15mp99mslist[(skynetpro_count | int / 2) | int] }}"
+        
+        # Averages
+        averagestoragespending: "{{ sumstoragespending | int / ansible_play_hosts | length }}"
+        averageuploadspending: "{{ sumuploadspending | int / ansible_play_hosts | length }}"
+        averagedownloadspending: "{{ sumdownloadspending | int / ansible_play_hosts | length }}"
+        averagefundaccountspending: "{{ sumfundaccountspending | int / ansible_play_hosts | length }}"
+        averagecontractfees: "{{ sumcontractfees | int / ansible_play_hosts | length }}"
+        averagetransactionfees: "{{ sumtransactionfees | int / ansible_play_hosts | length }}"
+        averageaccountbalancecost: "{{ sumaccountbalancecost | int / ansible_play_hosts | length }}"
+        averagefundaccountcost: "{{ sumfundaccountcost | int / ansible_play_hosts | length }}"
+        averageupdatepricetablecost: "{{ sumupdatepricetablecost | int / ansible_play_hosts | length }}"
+        averagetotalallocated: "{{ sumtotalallocated | int / ansible_play_hosts | length }}"
+        averageunspent: "{{ sumunspent | int / ansible_play_hosts | length }}"
+        averagepreviousspending: "{{ sumpreviousspending | int / ansible_play_hosts | length }}"
+        averagetotalspent: "{{ sumtotalspent | int / ansible_play_hosts | length }}"
+        averageunspentallocated: "{{ sumunspentallocated | int / ansible_play_hosts | length }}"
+        averageunspentunallocated: "{{ sumunspentunallocated | int / ansible_play_hosts | length }}"
+        averagedownloadscost: "{{ sumdownloadscost | int / ansible_play_hosts | length }}"
+        averageuploadscost: "{{ sumuploadscost | int / ansible_play_hosts | length }}"
+        averageregistryreadscost: "{{ sumregistryreadscost | int / ansible_play_hosts | length }}"
+        averageregistrywritescost: "{{ sumregistrywritescost | int / ansible_play_hosts | length }}"
+        averagerepairdownloadscost: "{{ sumrepairdownloadscost | int / ansible_play_hosts | length }}"
+        averagerepairuploadscost: "{{ sumrepairuploadscost | int / ansible_play_hosts | length }}"
+        averagesubscriptionscost: "{{ sumsubscriptionscost | int / ansible_play_hosts | length }}"
+      run_once: True
+
+    - name: Set H in SC, KS constants
+      set_fact:
+        hinsc: "{{ 1e+24 }}"
```

Comment:
nit: this might be a useful constant for other playbooks in the future, could just add to a var file.

Reply:
Which one? Although I sure hope it's never going to be useful cause I don't want ansible playbooks to be our go-to solution for these things.

Reply:
`hinsc`, this might help with future allowance of wallet maintenance for example.

Reply:
I was wondering what file to put it into but I went with all.yml for now.

Reply:
oh haha sorry yea that is fine.